packages/tokens/build.js

```diff
@@ -249,6 +255,38 @@ StyleDictionary.registerFormat({
   },
 })
 
+/** Formats declarations exports for composite tokens */
+StyleDictionary.registerFormat({
+  name: 'typescript/es6-declarations/composite',
+  formatter: function ({ dictionary, options }) {
+    let tokens = dictionary.allTokens,
+      declaration = '`export declare const ${options.exportName}: {\n'
```

Comment:
Looks like a last minute change broke things, getting the following error running `yarn tokens:build` against the branch:
```
dist/types/font/styles.d.ts
[error] dist/types/font/styles.d.ts: SyntaxError: Unterminated template literal. (82:2)
[error]   80 |     letterSpacing: number
[error]   81 |   }
[error] > 82 | }
[error]      |  ^
```

that looks related to the ` mark here turning the whole file into a string except the `${options.exportName}` bit.

Reply:
```suggestion
      declaration = `export declare const ${options.exportName}: {\n`
```